lua/carbon-now.lua

```diff
@@ -91,37 +103,47 @@ local function get_open_command()
   if vim.fn.has("win32") then
     return "start"
   end
+
+  vim.api.nvim_err_writeln("[carbon-now.nvim] Couldn't find a launch command")
+  return "echo"
 end
 
-local function create_snippet(opts)
+---@param opts {args: string, line1: integer, line2: integer}
+local function create_snapshot(opts)
```

Comment:
any reason to change the function name?

Reply:
It's a typo. Snippet is a relative small amount of code used to excerpt into a larger codebase (autocompletion). Meanwhile the Snapshot, is a photograph, especially one taken quickly or in a given moment,to keep track of state or quick share.

Reply:
makes sense, but since english is not my native language, we usually use the word "snippet" for both cases.